client/src/pages/CreateOrganizationProfile.js

```diff
@@ -138,8 +143,10 @@ const CreateOrgProfile = (props) => {
           });
         }
       } else {
-        alert("You must be logged in to create an organization profile");
-        return;
+        return createOrganizationFormDispatch({
+          type: CREATE_ORGANIZATION_ERROR,
+          error: `You must be logged in to create an organization profile`,
+        });
```

Comment:
I find this unnecessary though, as the user will not be able to even get to the point of trying to create an org profile without proper authentication nor having an user profile.

Reply:
Let's clean up later